src/flow-spec/IBlock.ts

```diff
@@ -190,7 +190,14 @@ export function createEvalContactFrom(contact: IContact): IContact {
 }
 
 export function evaluateToBool(expr: string, ctx: object): boolean {
-  return JSON.parse(evaluateToString(expr, ctx).toLowerCase())
+  const result = evaluateToString(expr, ctx).toLowerCase().trim()
```

Comment:
Is there a measurable performance impact in making this change, and if so, what is that measurement?